### ISSUE TITLE
Tailwind is used as CDN only recommended for development

### ISSUE THREAD

**Issue by FJLendinez** [OpenUnited/platform#52]
I was inspecting the project because I love Django + HTMX and your codebase is so wonderful, I appreciate the contribution to the open source.

## The problem

I saw that in base.html, tailwind is inserted from CDN. Attending the documentation they advice that this is only for development purposes and they recommends another way to use for production.

## The solution

Tailwind creates "Tailwind standalone", an executable that "build" the minified tailwind required without using node or similar. On the other hand, someone creates the python package "[pytailwindcss](https://github.com/timonweb/pytailwindcss)" that install "Tailwind standalone" using the python package manager.

### Step 1: Install

```shell
pip install pytailwindcss
```

### Step 2: Initialize

```shell
tailwindcss init
```

### Step 3: Configure

In tailwind.config.json

```js
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ['templates/*.html', 'templates/**/*.html', '**/templates/**/*.html'],
  theme: {
    extend: {},
  },
  plugins: [],
}
```

### Step 4: Build

When you decide where to start building (in CI, in precommit, deploying...) you just need to execute:

```
tailwindcss -o ./static/tailwind.css --minify
```

In my case, I have a Makefile for development that has the following command:

```Makefile
cssdev:
	@tailwindcss -o ./static/tailwind.css --minify -w
.PHONY: cssdev
```
And then, when I am developing, I start tailwind to watch changes and build the minified css required.

```
make cssdev
```

> Rebuilding...
> 
> Done in 370ms.

I hope all of this help you to improve your project, keep doing this great job!

:)

**Comment by dogukanteber** [comment]
Thanks a lot for your feedback @FJLendinez. We are currently under heavy development because of that reason we are using CDN. However, we will benefit from this because personally, I did not know this approach and it is cleaner than using node_modules, in my opinion.

If you have time, you can open a PR to take credit for your work. Otherwise, I can follow the steps you proposed as a solution.

Once again, thank you! ❤️

**Comment by adrianmcphee** [comment]
Thanks!! Super helpful @FJLendinez

**Comment by adrianmcphee** [comment]
@dogukanteber pls include a fix for this in your next PR

**Comment by dogukanteber** [comment]
Sure thing 👍 @adrianmcphee

**Comment by dogukanteber** [comment]
Closed by #56
